mne/viz/evoked.py

```diff
@@ -1258,38 +1258,45 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
                       topomap_args=None):
     """Plot evoked data as butterfly plot and add topomaps for time points.
 
+    .. note:: Axes to plot in can be passed by the user through ``ts_args`` or
+    ``topomap_args``. In that case both ``ts_args`` and ``topomap_args`` axes
```

Comment:
You need to indent these lines. Typically we do:
```
.. note:: blah blah
          blah blah
```
But even this is probably enough:
```
.. note:: blah blah
    blah blah
```

Reply:
ok, thanks!